### ISSUE TITLE
Physics Character Controller Demo not working without INPUT define

### ISSUE THREAD

**Issue by JoelFrutiger** [Unity-Technologies/EntityComponentSystemSamples#194]
In order for the Character Chontroller demo to work i had to add the UNITY_INPUT_SYSTEM_EXISTS to the defines. Maybe this should be added per default in the ProjectSettings.asset:

scriptingDefineSymbols:
    1: UNITY_POST_PROCESSING_STACK_V2;UNITY_INPUT_SYSTEM_EXISTS

**Comment by lawrnce** [comment]
@JoelFrutiger Thanks!

**Comment by AwokeKnowing** [comment]
I add it but it immediately disappears. how can i add it? (linux)

**Comment by MyelinsheathXD** [comment]
Hey! 
try  #if UNITY_INPUT_SYSTEM_EXISTS

**Comment by AwokeKnowing** [comment]
@myeg yeah i see in the code there are these #if UNITY_INPUT_SYSTEM_EXISTS  and I did add that thing on the scripting define symbols, and that makes the mouse work, but not the keyboard.

When using keyboard debugger, i see for example when I press w is says 'character w' and when i press s  it says 'format=KEYS' s
and in general it's kind of wacko with the mappings.   it seems Unity has made some wrong assumption and is relying on some system service it shouldn't, or relying on some mapping that is different for windows vs linux.

But ubuntu 20.04 is current LTS and so it's quite sad that the unity input system doesn't work.
if i use getKey  directly it works fine.  So it's something about the implementation of the input system which tries to be more abstract and universal yet comically fails at it's most core purpose to support WSAD/space/arrows

**Comment by JannikStaub** [comment]
If you are not using the new Input System and dont want to enable this scripting define you can paste this code to the start of the DemoInputGatheringSystem OnUpdate()

#if !UNITY_INPUT_SYSTEM_EXISTS
        m_CharacterMovement = new Vector2(UnityEngine.Input.GetAxisRaw("Horizontal"), UnityEngine.Input.GetAxisRaw("Vertical"));
        m_CharacterLooking = new Vector2(UnityEngine.Input.GetAxisRaw("Mouse X"), UnityEngine.Input.GetAxisRaw("Mouse Y"));
        m_CharacterFiring = UnityEngine.Input.GetMouseButton(0) ? 1 : 0;
        m_CharacterJumped = UnityEngine.Input.GetKey(KeyCode.Space);
#endif
